app/views/splash/index.html.erb

```diff
@@ -39,6 +45,7 @@
             <%= image_tag("splash-logos/BMlogo.png", class: "tenant-logo", alt: "The British Museum") %>
           <% end %>
         </div>
+        </div>
```

Comment:
This appears to be an additional and unexpected closing `div`.  Am I reading that right?  Prior to this line you introduced 2 opening and closing div tags.  This one appears to be a bonus.